homeworks/serhii.noskov_SerhiiNSN/GamesProject/scripts/games.js

```diff
@@ -0,0 +1,207 @@
+        release_date: '2006-12-11',
+        profile_url: 'https://www.mmobomb.com/darkorbit',
+    },
+    {
+        id: 1122,
+        title: 'MultiVersus',
+        thumbnail: 'https://www.mmobomb.com/g/1122/thumbnail.jpg',
+        short_description: 'Match up in 1v1, 2v2 co-op, or 4-person free-for-all modes in this free-to-play Smash-Style Brawler!',
+        game_url: 'https://www.mmobomb.com/open/multiversus',
+        genre: 'Fighting',
+        platform: 'PC (Windows)',
+        publisher: 'Warner Bros. Games',
+        developer: 'Player First Games',
+        release_date: '2022-07-19',
+        profile_url: 'https://www.mmobomb.com/multiversus',
+    },
+    {
+        id: 5,
+        title: 'Crossout',
+        thumbnail: 'https://www.mmobomb.com/g/5/thumbnail.jpg',
+        short_description: 'Trick out your ride and take to the post-apocalyptic roads for battle in Crossout, the free-to-play vehicular combat game from Gaijin Entertainment! Featuring a vehicle design system with endless customization and fast-paced, armor-crunching combat, Crossout offers high-octane excitement in brief and explosive matches.',
+        game_url: 'https://www.mmobomb.com/open/crossout',
+        genre: 'Shooter',
+        platform: 'PC (Windows)',
+        publisher: 'Targem',
+        developer: 'Gaijin',
+        release_date: '2017-05-30',
+        profile_url: 'https://www.mmobomb.com/crossout',
+    },
+];
+
+function init() {
+    const template = document.querySelector('[data-type="card-template"]');
+    let gamesDisplayed = [...gamesNew];
+    let checkboxNew;
+    let checkboxOld;
+    const radioButtons = document.querySelectorAll('.custom-radio');
+    const searchInput = document.querySelector('.search');
+    const dropList = document.querySelector('.drop-list');
+
+    function createCardElement(game) {
+        const cardCopy = document.createElement('li');
+        cardCopy.classList.add('games__card');
+        cardCopy.innerHTML = template.innerHTML;
+
+        cardCopy.querySelector('[data-card-img]').src = game.thumbnail;
+        cardCopy.querySelector('[data-card-title]').textContent = game.title;
+        cardCopy.querySelector('[data-card-release]').innerHTML = `<span class='bold'>Release_date:</span> ${game.release_date}`;
+
+        return cardCopy;
+    }
+
+    function renderCards(container, games) {
+        const fragment = document.createDocumentFragment();
+        const containerElement = document.querySelector(`[data-type="${container}"]`);
+
+        games.forEach((game) => {
+            const card = createCardElement(game);
+            fragment.appendChild(card);
+        });
+
+        containerElement.appendChild(fragment);
+    }
+
+    function handleCheckboxChange() {
+        const isNewChecked = checkboxNew.checked;
+        const isOldChecked = checkboxOld.checked;
+
+        radioButtons.forEach((radioButton) => {
+            radioButton.disabled = isNewChecked || isOldChecked;
```

Comment:
I mean, that if a filter is not working, you should show it to a user. And add `disabled` to HTML. Cause these filters don't work always not by some conditions.
Also, would be good UX, if you add `cursor: not-allowed;` to label of disabled inputs